dbms/src/Flash/executeQuery.cpp

```diff
@@ -52,7 +52,8 @@ void prepareForExecute(Context & context)
     context.setQueryContext(context);
 
     QuotaForIntervals & quota = context.getQuota();
-    quota.addQuery(); /// NOTE Seems that when new time interval has come, first query is not accounted in number of queries.
+    quota
+        .addQuery(); /// NOTE Seems that when new time interval has come, first query is not accounted in number of queries.
```

Comment:
looks not reasonable, can we do not count comment into line length?

Reply:
I search for this issue. Unfortunately, it's no way to make `clang-format` ignore the length of comments.
Only two things we can do:
1. turn `clang-format` off for this code.
2. set `ReflowComments` to true then this line of code will be look like
```
quota.addQuery(); /// NOTE Seems that when new time interval has come, first query is not accounted in number of
                  /// queries.
```
Seems ok. But I don't know if it's good enough for all the other comments.